lab_main.py

```diff
@@ -0,0 +1,138 @@
+"""importing module enum"""
+import enum
+
+
+class Type(enum.Enum):
+    """
+    Creating class for clothing's types
+    """
+    SHIRT = 1
+    JEANS = 2
+    JACKET = 3
+    DRESS = 4
+
+
+class Clothing:
+    """
+    Creating class Clothing for clothing elements
+    """
+
+    def __init__(self, name, description, location, colour, size, types):  # pylint: disable=too-many-arguments
+        self.name = name
+        self.description = description
+        self.location = location
+        self.colour = colour
+        self.size = size
+        self.type = types
+
+    def get_name(self):
+        """
+        Getter for name
+        :return self.name
+        """
+        return self.name
+
+    def get_description(self):
+        """
+        Getter for description
+        :return self.description
+        """
+        return self.description
+
+    def get_location(self):
+        """
+        Getter for location
+        :return self.location
+        """
+        return self.location
+
+    def get_colour(self):
+        """
+        Getter for colour
+        :return self.colour
+        """
+        return self.colour
+
+    def get_size(self):
+        """
+        Getter for size
+        :return self.size
+        """
+        return self.size
+
+    def get_type(self):
+        """
+        Getter for type
+        :return self.type
+        """
+        return self.type
+
+
+class Wardrobe:
+    """ Creating class Wardrobe for list of clothing"""
+
+    def __init__(self):
+        self.list_clothing = []
+
+    def add_clothing(self, cloth):
+        """
+        Adding cloth to list
+        :return None
+        """
+        self.list_clothing.append(cloth)
+
+    def go_out(self):
+        """
+        Whether you can go out or not
+        :return number, out
+        """
+        temp_set = set()
+        out = False
+        for cloth in self.list_clothing:
+            temp_set.add(cloth.get_type())
+        number = len(temp_set)
+        if number > 3:
+            print(f'Number of clothing: {number}. You can go out')
+            out = True
+        else:
+            print(f'Number of clothing: {number}. You can\'t go out')
+        return number, out
+
+    def sort(self):
```

Comment:
function name should incorporate the information about sort type, for ex:  sort_by_size()